Makefile

```diff
@@ -63,12 +63,12 @@ ifeq ($(SGX_DEBUG), 1)
 	# we build with cargo --release, even in SGX DEBUG mode
```

Comment:
I think this is wrong now, isn't it?

Reply:
Yes, I removed the obsolete comments in [8d7555a](https://github.com/integritee-network/worker/pull/1200/commits/8d7555acd1c2036a6d22782f4a12713684c29af9).

Reply:
Perfect, thanks!